pkspace/utils/trainer.py

```diff
@@ -0,0 +1,2 @@
+def train(spaces, answers, model):
```

Comment:
@rstevanak Is this function really necessary? It currently does not seem that it would add much functionality to the `model.fit` call below.